### ISSUE TITLE
Ability to set SRID in to_wkb/wkt function

### ISSUE THREAD

**Issue by jorisvandenbossche** [pygeos/pygeos#137]
Assume you want to write EWKB/EWKT that includes SRID information (eg for inserting into Postgis). This is currently already supported, as you can do:

```
geoms = pygeos.points(...)
geoms_with_srid = pygeos.set_srid(geoms, 4326)
pygeos.to_wkb(geoms_with_srid, include_srid=True)
```

However, the above means you need to create a full copy of the array of geometries, just for getting the EWKB (assuming you don't need to set the SRID for anything else), which can be wasteful when working with a large array of geometries. 
In principle, we could also clone and set the srid on the fly within the `to_wkb` ufunc. This does not avoid the cloning of all geometries, but avoids keeping those clones alive for the full array, which should improve memory usage.

**Comment by caspervdw** [comment]
Good point. I do have the feeling that it is more logical to set the SRID at geometry construction, and not at the IO. Or maybe we want both features.

**Comment by jorisvandenbossche** [comment]
To be clear, I wouldn't remove the `set_srid`, and there might be *some* use case for being able to specify the srid in constructive methods (not directly a use case myself though). I agree that in general it is more logical to have this functionality at construction time.

But here in `to_wkb`, it is mainly to avoid the memory overhead of requiring a copy of the full array. And since, in practice, setting the srid for converting to WKB is a typical use case (I think), it seems nice to just be able to specify it at IO time, if you only need it then.
